sample/ERC000038.json

```diff
@@ -0,0 +1,589 @@
+{
+  "id": "ERC000038",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": [
+    "attributes"
+  ],
+  "description": "Shellfish contextual information associated with molecular data. The checklist has been developed in collaboration with EMBRIC Project partners.",
+  "title": "ENA Shellfish Checklist",
+  "author": "ENA",
```

Comment:
As the author should it be this _JSON schema author_, or the _original checklist author_? On the other schemas we are assuming the JSON schema author.